Sources/Container/List/ListViewController.swift

```diff
@@ -22,7 +22,7 @@ import UIKit
 /// let fruitListViewController: FruitListViewController = .instantiate()
 /// fruitListViewController.model = .init(items: [FruitViewModel(...), ...])
 /// ```
-public final class ListViewController<View: StatefulViewProtocol>: StatefulViewController<ListViewModel<View.Model>>, UITableViewDataSource {
+open class ListViewController<View: StatefulViewProtocol>: StatefulViewController<ListViewModel<View.Model>>, UITableViewDataSource {
```

Comment:
Please make all overridable methods also `open`. There are `loadView`, `viewDidLoad` and `didChangeModel`.

Reply:
Done.